sources/packages/backend/apps/api/src/route-controllers/note/note.aest.controller.ts

```diff
@@ -79,7 +80,7 @@ export class NoteAESTController extends BaseController {
   @Get("institution/:institutionId")
   async getInstitutionDetails(
     @Param("institutionId", ParseIntPipe) institutionId: number,
-    @Query("noteType") noteType: NoteType,
+    @Query("noteType", new ParseEnumQueryPipe(NoteType)) noteType: NoteType,
```

Comment:
`noteType` is probably nullable here, right?

Reply:
ah.. yes. 👍